### DISCUSSION TITLE
Prisma in Docker with Bun?

### DISCUSSION THREAD

**Discussion by mrhappyma** [prisma/prisma#22360] [Q&A]
### Question

I'm having a lot of trouble getting prisma working in a docker container, while running TS with [bun](https://bun.sh). Has anyone had any success doing this (and can share a Dockerfile)?

**Comment by ludralph** [comment]
Hi @mrhappyma 👋 

There is an open issue where users ran into issues when using Prisma in Docker with Bun.

- https://github.com/prisma/prisma/issues/21241

There are some workarounds in the issue. Can you please take a look and let us know if any of the solutions work for you?

**Reply by mrhappyma** [reply]
I was able to get `prisma generate` working by adding nodejs, but my code hangs on the first db query (not when creating the client).

**Reply by brianrumburg** [reply]
I experienced this same issue today.  The Prisma client was hanging on first query.  No error.  No timeout.  Using the debian bun base image fixed it.  Try using:

`FROM oven/bun:1-debian`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
